src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1059,12 +1059,13 @@ private static function processObject($object)
 
 		$location = JsonLD::fetchElement($object, 'as:location', 'as:name', '@type', 'as:Place');
 		$location = JsonLD::fetchElement($location, 'location', '@value');
-
-		// Some AP software allow formatted text in post location, so we run all the text converters we have to boil
-		// down to HTML and then finally format to plaintext.
-		$location = Markdown::convert($location);
-		$location = BBCode::convert($location);
-		$location = HTML::toPlaintext($location);
+		if ($location) {
```

Comment:
I prefer a check with `empty(...)` but this surely is cosmetical and something one can have different opinions about it :-)

Reply:
Just "cosmetic" is fine. And none of the values covered by falsy check are concerned by text conversion:
- `false`
- `null`
- `""`
- `0`, `0.0`, `-0`, `-0.0` and `"0"`
- `[]`